crates/re_sdk/src/recording_stream.rs

```diff
@@ -908,20 +921,32 @@ impl RecordingStream {
     /// [SDK Micro Batching]: https://www.rerun.io/docs/reference/sdk-micro-batching
     /// [component bundle]: [`AsComponents`]
     #[inline]
-    pub fn log_timeless(
+    pub fn log_static(
         &self,
         ent_path: impl Into<EntityPath>,
         arch: &impl AsComponents,
     ) -> RecordingStreamResult<()> {
-        self.log_with_timeless(ent_path, true, arch)
+        self.log_with_static(ent_path, true, arch)
+    }
+
+    #[deprecated(since = "0.16.0", note = "use `log_static` instead")]
+    #[doc(hidden)]
+    #[inline]
+    pub fn log_with_timeless(
+        &self,
+        ent_path: impl Into<EntityPath>,
+        statically: bool,
```

Comment:
As mentioned in other PR, I think `static_` is a better name for avoiding keyword nameclash. `statically` sounds like it could be a different concept from `static`